app/controllers/api/templates_controller.rb

```diff
@@ -1,11 +1,16 @@
 module Api
   class TemplatesController < BaseController
+    include Api::Mixins::Genealogy
     include Subcollections::Policies
     include Subcollections::PolicyProfiles
     include Subcollections::Tags
 
-    def edit_resource(type, id = nil, data = {})
-      super(type, id, data.extract!('name', 'description'))
+    VALID_EDIT_ATTRS = %w[description custom_1 name child_resources parent_resource].freeze
```

Comment:
custom_1 should not be needed here because vm_custom_attrs will take care of all of the custom_# columns